src/assets/data/studyYears.js

```diff
@@ -0,0 +1,13 @@
+const studyYearsList = [
```

Comment:
We might want to include general terms like 1st Year/Freshman instead of just 1A/B for the out-of-town students

Reply:
Yeah I was debating about what to name these. Maybe just 1st Year/2nd Year etc. is good?